resources/ts/Pages/Integrations/Index.tsx

```diff
@@ -168,17 +198,22 @@ const Index = ({ integrations, paginationInfo, credentials }: Props) => {
       </div>
       {integrations.length > 0 && (
         <ul className="flex flex-col w-full gap-9">
-          {integrations.map((integration) => (
-            <li className="flex w-full" key={integration.id}>
-              <IntegrationCard
-                {...integration}
-                credentials={credentials}
-                onEdit={(id) =>
-                  router.get(`${translateRoute("/integrations")}/${id}`)
-                }
-              />
-            </li>
-          ))}
+          {integrationsWithCredentials.map(
+            ({ credentials, ...integration }) => (
+              <li className="flex w-full" key={integration.id}>
+                <IntegrationCard
+                  {...integration}
+                  auth0TestClient={credentials.auth0TestClient}
+                  auth0ProdClient={credentials.auth0ProdClient}
+                  uiTiDv1TestConsumer={credentials.uiTiDv1TestConsumer}
+                  uiTiDv1ProdConsumer={credentials.uiTiDv1ProdConsumer}
```

Comment:
Would maybe keep those in an object, you can ensure the entries in it are correct through the `Props` of the `IntegrationCard` and it would avoid having to destructure -> restructure the integration.